fixtures/attribute-behavior/src/App.js

```diff
@@ -282,12 +284,12 @@ function getRenderedAttributeValue(
   try {
     let container = createContainer();
     renderer.render(react.createElement(tagName, baseProps), container);
-    defaultValue = read(container.firstChild);
+    defaultValue = read(container.lastChild);
```

Comment:
Consistent with how we read the value of server-rendered markup. `firstChild` returns the DOCTYPE now if we render into a `document` on the client.